pkg/volume/configmap/configmap.go

```diff
@@ -156,6 +157,14 @@ func (b *configMapVolumeMounter) SetUpAt(dir string, fsGroup *int64) error {
 		return err
 	}
 
+	//reload the pod
```

Comment:
That's a valid point, but this whole reloading actually doesn't do what it was supposed to, I forgot to remove it. 

The IP was only accessible in pkg/kubelet/dockertools/docker_manager.go and pkg/kubelet/rkt/rkt.go (which bothers me a little, because this means that the IP is not stored in pod data persistently at this point).